3.analyze_data/compare_raw_mitocheckjump.ipynb

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
<p>This looks very similar to Mike's coding style</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/JUMP-single-cell/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>